### DISCUSSION TITLE
How to publish release ?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/react-native-windows#10445] [Q&A] [answered]
Create App Packages Release is successful but when I launch the app its shows a metro error!

![image](https://user-images.githubusercontent.com/65512279/186775776-7adc5547-445c-4bb6-bc2b-e658a407fd02.png)

**Comment by chrisglein** [comment, marked as answer]
We have some detailed documentation of the steps for publishing written up here: https://microsoft.github.io/react-native-windows/docs/app-publishing

Make sure you're creating a Release package, not a Debug package (which would still have Metro running).
